jupyter/launch-jupyter-interface.sh

```diff
@@ -3,7 +3,7 @@
 # This script sets SSH tunnel and opens Chrome Browser with SOCKS proxy
 # configured for direct access to the Jupyter cluster.
 
-set -euxo pipefail
+set -euo pipefail
```

Comment:
Any reason to remove `-x`?

Reply:
One actually - readability. This script is used locally to connect to Jupyter via SOCKS proxy and with `-x` all commands were additionally printed which doesn't really makes sense when you use it:
```
$ ./launch-jupyter-interface.sh -h
+ DIR=.
+ [[ ! -d . ]]
+ readonly DIR
+ source ./../util/utils.sh
+ for i in '"$@"'
+ case $i in
+ usage
+ cat
Creates an SSH tunnel and socks proxy and launches Chrome, using the environment
variable DATAPROC_CLUSTER_NAME for the unique cluster name. The cluster metadata
must contain a value for the key 'JUPYTER_PORT'.

If the appropriate environment variables are not set and the appropriate command"
line arguments are not given, then the usage message will be displayed and the "
script will exit."

usage: ./launch-jupyter-interface.sh [-h] [-c=cluster-name] [-z=zone]"
  -h                 display help"
  -z=zone            specify cloud zone for cluster"
  -c=cluster-name    specify unique dataproc cluster name to launch"
+ exit 1
```
After removal, output is clean:
```
$ ./launch-jupyter-interface.sh -h
Creates an SSH tunnel and socks proxy and launches Chrome, using the environment
variable DATAPROC_CLUSTER_NAME for the unique cluster name. The cluster metadata
must contain a value for the key 'JUPYTER_PORT'.

If the appropriate environment variables are not set and the appropriate command"
line arguments are not given, then the usage message will be displayed and the "
script will exit."

usage: ./launch-jupyter-interface.sh [-h] [-c=cluster-name] [-z=zone]"
  -h                 display help"
  -z=zone            specify cloud zone for cluster"
  -c=cluster-name    specify unique dataproc cluster name to launch"
```